to_do_robot.rb

```diff
@@ -1 +1,109 @@
-puts "Hello!"
+require_relative 'to_do_item_class.rb'
+require_relative 'to_do_list_class.rb'
+puts "Welcome to your To-Do list!"
+
+class MakeToDo
+  def initialize
+    puts "Ready to create your to-do list?(Yes or No?)"
+    answer = gets.chomp.downcase
+
+    case answer
+    when 'yes'
+      puts "What would you like to name your ToDo list?"
+      name = gets.chomp.capitalize
+      @list = List.new(name)
```

Comment:
If you're always passing a name to the list, then the default value in the `List` class is probably unnecessary.